src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/gui/ProxyControlGui.java

```diff
@@ -255,6 +260,10 @@
 
     static final String PREFIX_HTTP_SAMPLER_NAME = "proxy_prefix_http_sampler_name"; // $NON-NLS-1$
 
+    static final String HTTP_SAMPLER_NUMBERING_MODE = "proxy_http_sampler_numbering_mode"; // $NON-NLS-1$
```

Comment:
Could this be an Enum? This would allow better safety and ease of extension if another mode is required?